scispacy/linking.py

```diff
@@ -0,0 +1,116 @@
+from spacy.tokens import Doc
+from spacy.tokens import Span
+
+from scispacy.candidate_generation import CandidateGenerator
+
+
+class EntityLinker:
+    """
+    A spacy pipeline component which identifies entities in text which appear
+    in a knowledge base.
+
+    Currently, there are two defaults: the Unified Medical Language System (UMLS) and
+    the Medical Subject Headings (MESH) dictionary.
+
+    Currently this implementation just compares string similarity, returning
+    entities above a given threshold.
+
+
+    This class sets the `._.umls_ents` attribute on spacy Spans, which consists of a
+    List[Tuple[str, float]] corresponding to the UMLS concept_id and the associated score
+    for a list of `max_entities_per_mention` number of entities.
+
+    You can look up more information for a given id using the umls attribute of this class:
+
+    print(linker.umls.cui_to_entity[concept_id])
```

Comment:
because the attribute is still called umls, this is a bit confusing now that mesh is an option. I'm guessing you left it as umls throughout for backward compat?